### PR TITLE
Mobile Navigation Menu

### PR DESCRIPTION
**Pull request by mirwaisfarahi**
In this section the Mobile Navigation Menu has been created for smartphones / tablets with CSS and JavaScript. It is a curtain menu in which all the navbar items are linked to the specific section of the page. 

The changes are pushed in 'created_mobile_menu_feature' branch.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -64,7 +71,7 @@ <h3>LET'S CONNECT</h3>
 
             <!-- Works Section -->
             <section class="works-section">
```
Comment (gtekle) [inline index.html]: - [x] When you click the `Portfolio` nav link, it should take you to the works section. It is also good to add a heading such as `Works` or `Projects` to this section.

File: index.html [hunk 2]
```diff
@@ -230,15 +237,24 @@ <h2 id="contact-me">Contact me </h2>
...
+
+    <script>
+        function openNavbar() {
+            document.getElementById('mobileNav').style.height = '100%';
+        }
+
+        function closeNavbar() {
+            document.getElementById('mobileNav').style.height = '0%';
+        }
+    </script>
```
Comment (gtekle) [inline index.html]: - [x] The best practice is to put JavaScript code in its own file and include the file in the HTML. Thus, kindly create new file and add JavaScript code in that file. [reference](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)
JS-best-practice

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @mirwaisfarahi :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
this-close

## Highlights
:heavy_check_mark: Clean code
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Nice UI

## Required Changes ♻️
_Check the comments under the review._

- [x] You should setup the correct linters for this project. You should use the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
correct-linters-to-setup
installed-linters


## Optional suggestions

- [ Optional ] It is good to make commit messages meaningful to give an idea about the changes that are made in that commit.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @mirwaisfarahi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- README looks professional 👀.
- linters setup properly ✔️ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**